### ISSUE TITLE
[Proposal] Use github pages

### ISSUE THREAD

**Issue by clsource** [wireshell/wireshell-doc-page-php#1]
if the repo was called wireshell.github.io then only with an index.html file the page will be hosted on github servers :) , later it can be added the domain .pw :+1:

**Comment by marcus-herrmann** [comment]
I plan to use partials since the documentation is ever growing. A single index.html (as the index.php is now) will continue to be confusing. Will look in to Jekyll, Metalsmith.io et al in the future

**Comment by clsource** [comment]
yes indeed a single file is difficult to mantain. how a bout using gitbook
https://www.gitbook.com/

the styles could be changed :+1:

**Comment by clsource** [comment]
So I made a testing repo

http://wireshell.github.io/
https://github.com/wireshell/wireshell.github.io

there you can see that the root index.html can contain custom html code
and the documentation is in the _book directory.

The main benefit is it enables you for ouput the documentation in epub, pdf and so on
using the gitbook app. Also all the files are just markdown formatted. :+1:

**Comment by clsource** [comment]
So I extracted the static info of the site, the ones that will not change in the near future and created a new index.html file. All the docs will be in the gitbook formatted directory named _book, so it can be easily translated and converted to other formats :hamburger: :fireworks:

**Comment by clsource** [comment]
should be this repo deleted?

**Comment by marcus-herrmann** [comment]
Yes
